src/plcc/load_spec/validate_spec/validate_syntactic_spec/check_parsing_table_test.py

```diff
@@ -0,0 +1,36 @@
+from pytest import raises, mark, fixture
+from collections import defaultdict
+
+from .check_parsing_table_for_ll1 import check_parsing_table_for_ll1, LL1Error
+
+def test_incorrect_example_table():
```

Comment:
What does "incorrect" and "correct" mean? Encode this in the test name.

Assuming "incorrect" means "a cell with more than one entry", then a good name might be `test_more_than_one_entry_yields_error`.

Please update the other names in this file too.